modules/installation-creating-azure-bootstrap.adoc

```diff
@@ -41,11 +41,20 @@ support with your installation logs.
 this topic and save it as `04_bootstrap.json` in your cluster's installation directory. This template
 describes the bootstrap machine that your cluster requires.
 
-. Export the following variables required by the bootstrap machine deployment:
+. Export the bootstrap ignition variable:
++
+[source,terminal]
+----
+$ BOOTSTRAP_URL_EXPIRY=`date -u -d "10 hours" '+%Y-%m-%dT%H:%MZ'`
+----
++
+[source,terminal]
```

Comment:
prefer added them in the same [source, terminal]
bootstrap_url_expiry=...
export BOOTSTRAP_URL=...
export BOOTSTRAP_IGNITION=...

ref https://github.com/openshift/installer/blob/master/docs/user/azure/install_upi.md#launch-the-temporary-cluster-bootstrap

Reply:
To take advantage of the copy functionality on the Docs site, official docs guidelines strongly encourage individual commands be broken into separate code blocks.

Reply:
Good to know. That makes sense.